source/Projectile.cpp

```diff
@@ -287,35 +295,54 @@ shared_ptr<Ship> Projectile::TargetPtr() const
 
 void Projectile::CheckLock(const Ship &target)
 {
-	double base = hasLock ? 1. : .5;
+	double base = hasLock ? 1. : .15;
 	hasLock = false;
 	
-	// For each tracking type, calculate the probability that a lock will be
-	// lost in a given five-second period. Then, since this check is done every
-	// second, test against the fifth root of that probability.
+	// For each tracking type, calculate the probability twice every second that a
+	// lock will be lost
 	if(weapon->Tracking())
 		hasLock |= Check(weapon->Tracking(), base);
 	
 	// Optical tracking is about 15% for interceptors and 75% for medium warships.
```

Comment:
Would these numbers not change because of the change in the probability equation?

Reply:
I think these were always sort of a guesstimate. :man_shrugging:

Reply:
These numbers are kind of right- the check is against probability^0.2, so if the check runs 5 times the chance of the missile passing all of them and tracking for 5 seconds straight(on average) is (probability^0.2)^5, which reduces to just the probability. Checking twice as often would make it 2.5 seconds for 5 checks on average.